src/main/java/net/pms/dlna/MediaMonitor.java

```diff
@@ -206,49 +206,44 @@ public void stopped(DLNAResource res) {
 						LOGGER.debug("An error occurred when dumping monitor file: " + e);
 					}
 
-					File watchedFile = new File(rf.getFile().getAbsolutePath());
+					File playedFile = new File(rf.getFile().getAbsolutePath());
 
 					if (fullyPlayedAction == 3) {
 						// Move the video to a different folder
-						String newDirectory = configuration.getFullyPlayedOutputDirectory();
-						if (!newDirectory.endsWith("\\")) {
-							newDirectory += "\\\\";
-						}
+						String newDirectory = FileUtil.appendPathSeparator(configuration.getFullyPlayedOutputDirectory());
 
-						try {
-							if (watchedFile.renameTo(new File(newDirectory + watchedFile.getName()))) {
-								LOGGER.debug("Moved {} because it has been fully played", watchedFile.getName());
-							} else {
-								LOGGER.debug("Moving {} failed, trying again in 3 seconds", watchedFile.getName());
+						if (playedFile.renameTo(new File(newDirectory + playedFile.getName()))) {
+							LOGGER.debug("Moved {} because it has been fully played", playedFile.getName());
+						} else {
+							LOGGER.debug("Moving {} failed, trying again in 3 seconds", playedFile.getName());
 
-								try {
-									Thread.sleep(3000);
-								} catch (InterruptedException e) {
-									LOGGER.error(null, e);
-								}
+							try {
+								Thread.sleep(3000);
 
-								if (watchedFile.renameTo(new File(newDirectory + watchedFile.getName()))) {
-									LOGGER.debug("Moved {} because it has been fully played", watchedFile.getName());
+								if (playedFile.renameTo(new File(newDirectory + playedFile.getName()))) {
+									LOGGER.debug("Moved {} because it has been fully played", playedFile.getName());
 								} else {
-									LOGGER.info("Failed to move {}", watchedFile.getName());
+									LOGGER.info("Failed to move {}", playedFile.getName());
 								}
+							} catch (InterruptedException e) {
+								LOGGER.warn("Abandoning moving of {} because the thread was interrupted, probably due to UMS shutdown", e.getMessage());
+								LOGGER.trace("", e);
+								Thread.currentThread().interrupt();
 							}
-						} catch (Exception e) {
-							LOGGER.info("Failed to move {} because {}", watchedFile.getName(), e.getMessage());
 						}
 					} else if (fullyPlayedAction == 4) {
 						try {
 							if (Platform.isLinux()) {
-								FreedesktopTrash.moveToTrash(watchedFile);
+								FreedesktopTrash.moveToTrash(playedFile);
 							} else {
-								FILE_UTILS.moveToTrash(Arrays.array(watchedFile));
+								FileUtils.getInstance().moveToTrash(Arrays.array(playedFile));
 							}
 						} catch (IOException | FileUtil.InvalidFileSystemException e) {
-							LOGGER.error(String.format("Failed to send the file %s to the trash after it has been fully played. Please enable the functionality on your operating system.", watchedFile.getAbsoluteFile()));
-							LOGGER.trace("The error was", e);
```

Comment:
As previously described, this is not only determined by the OS and the message was misleading.